package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Authenticate with S3O via middleware",
   "main": "index.js",
   "scripts": {
-    "test": "npm test"
+    "test": "mocha --harmony test/**/*.spec.js"
```

Comment:
@JakeChampion To get past the unsupported block scoped issue; I noticed you changed the env to node 6, which might also work. Apologies, I think I nuked that change — let me revert it.